sci-visualization/gnuplot/gnuplot-6.0.0.recipe

```diff
@@ -73,7 +87,10 @@ BUILD()
 	aclocal
 	automake
 	runConfigure --omit-dirs binDir ./configure --bindir="$commandBinDir" \
+		--with-bitmap-terminals \
 		--without-cairo \
+		--with-lua=no \
+		--with-qt=qt6
```

Comment:
qt6 here and qt5 in the requirements?

Reply:
Allowed. Falls back to Qt5 if Qt6 not present during configure.

Reply:
Doesn't explain why you use Qt6 here.

Reply:
Fixed, in case the fallback gets removed in newer src revisions.